test/api/v3/integration/user/DELETE-user.test.js

```diff
@@ -30,6 +30,20 @@ describe('DELETE /user', () => {
     });
   });
 
+  it('does not need a password if logged in through Facebook', async () => {
+    await user.auth.facebook.id.del('/user', {
```

Comment:
yep, I figured that, (hence the WIP) I'm still looking at how I should use the requester to get the right id. I was looking at something else when I made this test. I'll update. Thanks!